drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-api/src/main/java/org/drools/workbench/screens/guided/dtable/shared/XLSConversionResult.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.guided.dtable.shared;
+
+import java.util.Objects;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+
+@Portable
+public class XLSConversionResult {
+
+    private String message;
+
+    public XLSConversionResult() {
+        this.message = "";
+    }
+
+    public XLSConversionResult(final String message) {
+        this.message = message;
+    }
+
+    public boolean isConverted() {
+        return Objects.equals("", message);
```

Comment:
This implementation suggests me `XLSCoversionResult` is more `XLSConversionError` rather than `Result`. Am I missing something?

Reply:
The main reason for this is just that it should in the future contain more than just the error.